presto-google-sheets/src/main/java/com/facebook/presto/google/sheets/SheetsConfig.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.google.sheets;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+public class SheetsConfig
+{
+    private String credentialsFilePath;
+    private String metadataSheetId;
+    private int sheetsDataMaxCacheSize = 1000;
+    private Duration sheetsDataExpireAfterWrite = new Duration(5, TimeUnit.MINUTES);
+
+    @NotNull
+    public String getCredentialsFilePath()
+    {
+        return credentialsFilePath;
+    }
+
+    @Config("credentials-path")
```

Comment:
Could you document the configs in a doc and put them in the `presto-docs` folder?